sql/schema_changer.go

```diff
@@ -356,12 +356,12 @@ func (sc *SchemaChanger) purgeMutations(lease *TableDescriptor_SchemaChangeLease
 		// Publish() will increment the version.
 		return nil
 	}); pErr != nil {
-		return pErr
+		return pErr.GoError()
```

Comment:
hmm, good point. I think it's fine now as the returned error is just used as part of error message.

```
if errPurge := sc.purgeMutations(&lease); errPurge != nil {
	return roachpb.NewErrorf("error purging mutation: %s, after error: %s", errPurge, pErr)
}
```